frontend/app/pages/service/registration/ServiceRegistration.tsx

```
@@ -0,0 +1,245 @@
+  })
+
+  const onSubmit = (data: ServiceFormData) => {
+    console.log(data)
+    SuccessAlert("Serviço cadastrado com sucesso!")
+    reset()
+    setPreviewImage(null)
+  }
+
+  const handleImageChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (e.target.files && e.target.files[0]) {
+      const file = e.target.files[0]
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)